.github/workflows/package-apisix-runtime-deb-openresty-1.21.yml

```diff
@@ -0,0 +1,88 @@
+name: package apisix-runtime deb for debianbullseye-slim with openresty 1.21
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Tag name for the release'
+        required: true
+        default: ''
+
+jobs:
+  build:
+    timeout-minutes: 60
+    env:
+      BUILD_APISIX_RUNTIME_VERSION: 1.1.2
+      ARCH: amd64
```

Comment:
this env is useless? we get the arch by `matrix.platform.arch`

Reply:
removed it. it was previously used for some reason but wasn't removed. Now removed it